libraries/tlv-account-resolution/src/state.rs

```diff
@@ -174,12 +244,15 @@ impl ExtraAccountMetas {
 
     /// Resolve a program-derived address (PDA) from the instruction data
     /// and the accounts that have already been resolved
-    fn resolve_pda(
-        resolved_metas: &[AccountMeta],
+    pub fn resolve_pda<M>(
```

Comment:
Can we keep this private?
```suggestion
    fn resolve_pda<M>(
```

Reply:
I popped this open because I figured, with this being a more broadly available library, someone might want to use this function so they can just feed the instruction details to it and not have to write any of the logic, provided they already established configs

Reply:
I'd prefer to leave it private until it's needed